src/app/screens/Home/index.jsx

```diff
@@ -27,6 +30,17 @@ class Home extends React.Component {
   }
 
   componentDidMount() {
+    browser.tabs
+      .query({ active: true, lastFocusedWindow: true })
```

Comment:
why not `currentWindow` do you know what's the difference?

Reply:
Good catch! Actually haven't tested with currentWindow instead.

Reply:
Pushed a change.
Should be currentWindow: https://developer.chrome.com/docs/extensions/reference/tabs/#get-the-current-tab